crates/pathfinder/examples/generate_tree.rs

```diff
@@ -0,0 +1,175 @@
+use anyhow::Context;
+use stark_hash::{stark_hash, StarkHash};
+use std::cell::RefCell;
+use std::io::BufRead;
+use std::num::NonZeroUsize;
+
+fn main() -> anyhow::Result<()> {
+    let mut args = std::env::args();
+    let myself = args.next().unwrap_or_else(|| String::from("generate_tree"));
+    let any = args.next().is_some();
+
+    if any {
+        eprintln!("USAGE: {myself}");
+        eprintln!();
+        eprintln!("Reads stdin for whitespace separated tuples or triples of 0x prefixed starkhashes, where key is the first value.");
```

Comment:
What are the 2nd and optional 3rd value?

Reply:
Starkhashes, they get collapsed contract state hash style (hash chain). But the code supports N, just in case. I didn't update this to match the impl.

Reply:
Tried to make this better.